.github/workflows/shippable_builds.yml

```diff
@@ -4,6 +4,10 @@ on:
   workflow_call:
   workflow_dispatch:
     inputs:
+      releaseVersion:
+        description: "Release version to build"
+        required: true
+        default: "8.0b3"
```

Comment:
I'd avoid a default unless there are valid cases to just click go. Better force the user to enter a value.